src/containers/tutor-home-page/add-photo-step/AddPhotoStep.jsx

```diff
@@ -5,44 +5,68 @@ import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import DragAndDrop from '~/components/drag-and-drop/DragAndDrop'
 import FileUploader from '~/components/file-uploader/FileUploader'
+
 import { validationData } from '~/containers/tutor-home-page/add-photo-step/constants'
 import { useStepContext } from '~/context/step-context'
 import useBreakpoints from '~/hooks/use-breakpoints'
-
+import { imageResize } from '~/utils/image-resize'
 import { style } from '~/containers/tutor-home-page/add-photo-step/AddPhotoStep.style'
 
 const AddPhotoStep = ({ btnsBox }) => {
   const { t } = useTranslation()
   const { isMobile, isTablet, isLaptopAndAbove } = useBreakpoints()
   const { handleStepData, stepData } = useStepContext()
   const [errorMessage, setErrorMessage] = useState('')
+
   const { photo } = stepData
 
-  const photoSrc = photo[0] ? URL.createObjectURL(photo[0]) : ''
+  const handleAddPhoto = async ({ error, files }) => {
+    if (error) {
+      setErrorMessage(error)
+      return
+    }
+
+    if (!files.length) {
+      setErrorMessage('')
+      handleStepData('photo', files)
+      return
```

Comment:
Maybe we can use `useForm` (`src/hooks/use-form.jsx`) for this validation?

Reply:
I think `useForm` is not applicable here

FileUploader and DragAndDrop components are responsible for file validation, and they lift the result of validation up